tests/test_rabbitmq.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from testcontainers.rabbitmq import RabbitmqContainer
+import time
+import pika
+
+def test_declare_queue():
+    rabbitmq_container = RabbitmqContainer("rabbitmq:3-management")
```

Comment:
Why can we not use the standard `rabbitmq:lastest` tag here?

Reply:
because it does not work with `rabbitmq:latest` :)